src/OffsetsData.hpp

```diff
@@ -253,6 +253,9 @@ OFFSET_DEFAULT(GetSearchPath, 16, 16)
 // CGameRules
 OFFSET_DEFAULT(IsMultiplayer, 33, 34)
 
+// IClientRenderable
+OFFSET_DEFAULT(GetModel, 8, 9)
```

Comment:
Did you actually find this offset for Linux or did you just assume it was "Windows offset + 1"?
Please double check because the real offset is 8.

Reply:
i dont know why i literally didnt even bother to look at the sdk, its obvious it shouldnt be +1, fixed